src/saltext/vmware/modules/vmc_org_users.py

```diff
@@ -0,0 +1,441 @@
+        expandProfile=expand_profile,
+    )
+
+    return vmc_request.call_api(
+        method=vmc_constants.GET_REQUEST_METHOD,
+        url=api_url,
+        refresh_key=refresh_key,
+        authorization_host=hostname,
+        description="vmc_org_users.search",
+        params=params,
+        verify_ssl=verify_ssl,
+        cert=cert,
+    )
+
+
+def add(
+    hostname,
+    refresh_key,
+    org_id,
+    user_names,
+    organization_roles,
+    skip_notify=False,
+    custom_roles=None,
+    service_roles=None,
+    skip_notify_registration=False,
+    invited_by=None,
+    custom_groups_ids=None,
+    verify_ssl=True,
+    cert=None,
+):
+    """
+    Add users on a given org
+
+    Please refer the `Organization User Invitation documentation <https://developer.vmware.com/apis/csp/csp-iam/latest/csp/gateway/am/api/orgs/orgId/invitations/post/>`_ to get insight of functionality and input parameters
+
+    CLI Example:
+
+      .. code-block:: bash
+
+          salt <minion id> vmc_org_users.add hostname=console.cloud.vmware.com org_id=org_id  ...
+
+    hostname
+          The host name of CSP
+
+    refresh_key
+          API Token of the user which is used to get the Access Token required for VMC operations
+
+    org_id
+          The Id of organization for which user list is retrieved
+
+    skip_notify:boolean
+        (Optional) This will decide to skip the notification to user
+
+    user_names: Array list of String
+        (Required) Usernames
+
+    organization_roles: Array of BaseRoleBindingDto
+        (Required) Organization roles assigned to user, uniqueItems: true
+        Its mandatory while sending invitation to user
+
+        BaseRoleBindingDto
+            The organization roles that will be assigned to the member
+
+            name: String
+                  The role name
+
+            resource: String
+                  maxLength: 200
+                  minLength: 0
+                  The resource in which the role is scoped to.
+                  The resource will be embedded in the Access Token "perms" claim, as part of the role
+
+            expiresAt: integer(int64)
+                  The timestamp the role expires at (measured in number of seconds since 1/1/1970 UTC) example:3609941597
+
+    custom_roles: Array of CustomRoleBindingDto
+       (Optional) custom roles assign to a user
+        CustomRoleBindingDto
+            name: String
+                  The role name
+
+            resource: String
+                  maxLength: 200
+                  minLength: 0
+                  The resource in which the role is scoped to.
+                  The resource will be embedded in the Access Token "perms" claim, as part of the role
+
+            expiresAt: integer($int64)
+                  example: 3609941597
+                  The timestamp the role expires at (measured in number of seconds since 1/1/1970 UTC)
+
+    service_roles: Array list of ServiceRolesDto
```

Comment:
**suggestion** it's probably good to reference the API terms, but... This doesn't really mean anything. (And I could go on about "Dto")

It'd be something like:

```
service_roles: 
    List of service roles to attach to a user. Example:
    
    .. code::
    
         salt my_minion vmc_org_users.add test@example.com service_roles=[{"serviceDefinitionLink": "some service", "serviceRoles": [{...}, {...}]]
```

Or whatever format those should be in. Basically the users should be able to copy from the docs, make some changes to the particulars, and paste the exact value on the command line.

Reply:
Updated the details as required.

Reply:
note that the example isn't actually an example -- that is, I can't copy/paste the cli example and change my hostname & other config values and actually ~create~ invite a user.

Reply:
added the proper cli example